public/app/plugins/panel/timeseries/panelcfg.gen.ts

```diff
@@ -11,8 +11,17 @@
 import * as common from '@grafana/schema';
 
 export interface Options extends common.OptionsWithTimezones {
+  exemplars: ExemplarConfig;
```

Comment:
You can't edit files with `.gen` in the name, they are auto generated, take a look at the header of the file as to the location of the generating file.

Reply:
@gtk-grafana I haven't edited these files manually but executed `make gen-cue` which modified these files

Reply:
I don't see any changes to `public/app/plugins/gen.go` though, so its output shouldn't have changed? Is this branch is stale compared to main?

Reply:
Oh nvm, I didn't see the .cue changes I was looking for go files, sorry.